projects/plugins/boost/app/assets/src/js/features/critical-css/folding-element/folding-element.tsx

```diff
@@ -27,13 +29,21 @@ const FoldingElement: React.FC< PropTypes > = ( {
 		height: expanded ? height : 0,
 	} );
 
+	const handleOnExpand = () => {
+		const newIsEditing = ! expanded;
+		setExpanded( newIsEditing );
+		if ( onExpand ) {
+			onExpand( newIsEditing );
+		}
+	};
```

Comment:
I got confused with the name `newIsEditing`, I guess you wanted to write `newIsExpanded`. 

```suggestion
	const handleOnExpand = () => {
		const newValue = ! expanded;
		setExpanded( newValue );
		if ( onExpand ) {
			onExpand( newValue );
		}
	};
```